common/src/main/java/fudge/notenoughcrashes/stacktrace/ModIdentifier.java

```diff
@@ -110,6 +132,42 @@ private static Set<CommonModMetadata> identifyFromClass(String className, ModsBy
         }
     }
 
+    @Nullable
+    private static MixinMerged findMixinMerged(StackTraceElement element) {
+        try {
+            Class<?> clazz = Class.forName(element.getClassName());
+            // Walk through methods because we don't know parameter types
+            Method[] methods = clazz.getDeclaredMethods();
+            for (Method method : methods) {
+                if (method.getName().equals(element.getMethodName())) {
+                    MixinMerged mixinMerged = method.getAnnotation(MixinMerged.class);
+                    if (mixinMerged != null) {
+                        return mixinMerged;
+                    }
+                }
+            }
+        } catch (ClassNotFoundException ignored) {}
+
+        return null;
+    }
+
+    @NotNull
+    private static Set<CommonModMetadata> identifyFromMixin(IMixinInfo mixin) {
+        Set<CommonModMetadata> metadata = mixinConfigToMods.get(mixin.getConfig());
+        if (metadata != null) return metadata;
+
```

Comment:
Use `mixinConfigToMods.computeIfAbsent(mixin.getConfig()) { ... }` instead of checking and putting manually